bundle/edu.gemini.p2checker/src/main/scala/edu/gemini/p2checker/rules/general/GeneralRuleHelper.scala

```diff
@@ -0,0 +1,29 @@
+package edu.gemini.p2checker.rules.general
+
+import edu.gemini.spModel.core.{NonSiderealTarget, TooTarget, SiderealTarget, Target}
+
+import scalaz._, Scalaz._
+
+class GeneralRuleHelper {
+
+  def samePosition(a: Target, b: Target): Boolean =
+    (a, b) match {
+
+      // Sidereal targets must have identical coordinates, proper motion, redshift, and parallax
```

Comment:
Unclear, probably. The old implementation compared strings values for coordinates but exact valued for everything else. I'll change it to do the same thing. :chicken: